packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -69,7 +69,7 @@ export abstract class TopContent extends Content {
     return new TopContentId(this.contentType, this.id)
   }
 
-  cloneWithFollowUp(newFollowUp: FollowUp): this {
+  cloneWithFollowUp(newFollowUp: FollowUp | undefined): this {
```

Comment:
I think I'd go for `cloneWithFollowUp(newFollowUp?: FollowUp): this { ... }` instead